<?xml version="1.0" ?>
<launch>
    <arg name="init_x"/>
    <arg name="init_y"/>
    <arg name="init_z"/>
    <arg name="max_vel"/>
    <arg name="odometry_topic"/>

    <!-- <node pkg="fael_exploration" type="topo_exploration_node" name="topo_exploration_node">
        <remap from="/odometry" to="$(arg odometry_topic)"/>
        <remap from="/vehicle_execute" to="/path_execution/vehicle_execute"/>
        <remap from="/vehicle_stop" to="/path_execution/vehicle_stop"/>
        <remap from="/topo_planner" to="/topo_planner/topo_planner"/>

        <param name="frame_id" value="world"/>
        <param name="fael_exploration/init_motion_enable" value="false"/>
        <param name="fael_exploration/init_x" value="$(arg init_x)"/>
        <param name="fael_exploration/init_y" value="$(arg init_y)"/>
        <param name="fael_exploration/init_z" value="$(arg init_z)"/>
        <param name="fael_exploration/max_vel" value="$(arg max_vel)" />
        <param name="fael_exploration/replan_too_long_thre" value="1.0"/>
        <param name="fael_exploration/replan_periodic_thre" value="2.5"/>

    </node> -->

    <!-- statistics and data visualization -->
    <include file="$(find visualization_tools)/launch/visualization_tools.launch">
        <arg name="odometry" value="$(arg odometry_topic)"/>
    </include>

</launch>
